astropy/units/tests/test_format.py

```diff
@@ -569,3 +570,35 @@ def test_powers(power, expected):
     s = unit.to_string()
     assert s == expected
     assert unit == s
+
+
+@pytest.mark.parametrize('string,unit', [
+    ('\N{MICRO SIGN}g', u.microgram),
+    ('\N{GREEK SMALL LETTER MU}g', u.microgram),
+    ('g\N{MINUS SIGN}1', u.g**(-1)),
+    ('m\N{SUPERSCRIPT MINUS}\N{SUPERSCRIPT ONE}', 1 / u.m),
+    ('m s\N{SUPERSCRIPT MINUS}\N{SUPERSCRIPT ONE}', u.m / u.s),
+    ('m\N{SUPERSCRIPT TWO}', u.m**2),
+    ('m\N{SUPERSCRIPT PLUS SIGN}\N{SUPERSCRIPT TWO}', u.m**2),
+    ('m\N{SUPERSCRIPT THREE}', u.m**3),
+    ('m\N{SUPERSCRIPT ONE}\N{SUPERSCRIPT ZERO}', u.m**10),
+    ('\N{LATIN CAPITAL LETTER A WITH RING ABOVE}', u.angstrom),
+    ('\N{GREEK CAPITAL LETTER OMEGA}', u.ohm),
+    ('\N{OHM SIGN}', u.ohm), # deprecated but for compatibility
+    ('°C', u.deg_C),
+    ('°', u.deg),
+])
+def test_unicode(string, unit):
+    assert u_format.Generic.parse(string) == unit
+    assert u.Unit(string) == unit
```

Comment:
I don't see it in the tests - will I be able to write u.Å in my source code?

Reply:
Ah yes. At the moment `u.Å` and `u.Ω` work. Micro does not.

@mhvk should we add the micro sign?

Reply:
removed again according to @mhvk